sum-of-multiples.md

```diff
@@ -4,4 +4,12 @@ multiples of either 3 or 5, we get 3, 5, 6 and 9, 10, 12, 15, and 18.
 The sum of these multiples is 78.
 
 Write a program that can find the sum of the multiples of a given set of
-numbers. If no set of numbers is given, default to 3 and 5.
+numbers.
+
+This exercise is in the process of being reworked.
+In most tracks, the set of numbers must be explicitly provided.
+In some other tracks, if no set of numbers is given, it defaults to 3 and 5.
+We would like all tracks to require the set of numbers be explicit.
+
+If you would like to help out, please take a look at
```

Comment:
Seems a bit vague. Should I more specific to "If your track falls into the latter category, please take a look at..."?

On the other hand, being very open may invite more contributors...?